app/Http/Controllers/Backend/DashboardController.php

```diff
@@ -5,16 +5,85 @@
 use Illuminate\Http\Request;
 use App\Http\Requests;
 use App\Http\Controllers\Controller;
+use App\Models\Backend\Order;
+use DB;
 
 class DashboardController extends Controller
 {
     /**
      * Index admmin
      *
+     * @param string $condition this is of group by
+     *
+     * @return view index of admin
+     */
+    public function index($condition = 'DAYNAME')
+    {
+        // create array save data report
+        $data = array();
+
+        // check condition group by.
+        // only group by with condition = DAYNAME or MONTHNAME of created_at
+        // if condition != DAYNAME or MONTHNAME then group by with DAYNAME
+        if ($condition == 'DAYNAME' or $condition == 'MONTHNAME') {
```

Comment:
[rank B] Please avoid hardcode, DAYNAME or MONTHNAME should be defined as constants.